components/Text-Utlities/Text-Utilities.html

```diff
@@ -0,0 +1,160 @@
+                   style="width:90%; height:150px; border:0; transform: scale(1); overflow:hidden;"
+                   sandbox="allow-scripts allow-same-origin">
+             </iframe>
+                <h3 class="sub_heading">Example of Different-Text-Sizes</h3>
+                <div>These are the examples of different text of different sizes.Include these classes <span style="color:var(--primary-color)">xl-text
+                lg-text-md-text-sm-text etc</span> to use them.</div>
+                <div class="typo_container"> 
+                        <p class="xl-text">This is extra large size text.</p>
+                        <p class="lg-text">This is large size text.</p>
+                        <p class="md-text">This is medium size text.</p>
+                        <p class="sm-text">This is small size text.</p>
+                        <p class="esm-text">This is extra small size text.</p>
+                </div>
+                <iframe
+                 src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=base16-dark&wt=none&l=auto&width=912&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Anonymous+Pro&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522xl-text%2522%253EThis%2520is%2520extra%2520large%2520size%2520text.%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522lg-text%2522%253EThis%2520is%2520large%2520size%2520text.%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522md-text%2522%253EThis%2520is%2520medium%2520size%2520text.%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522sm-text%2522%253EThis%2520is%2520small%2520size%2520text.%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522esm-text%2522%253EThis%2520is%2520extra%2520small%2520size%2520text.%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520"
+                 style="width:90%; height:150px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can also make class for this style as well.

Reply:
okay